components/src/core/hero/hero-banner.component.ts

```diff
@@ -11,6 +11,9 @@ import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@a
         <mat-divider class="pxb-hero-banner-divider" *ngIf="divider"></mat-divider>
     `,
     styleUrls: ['./hero-banner.component.scss'],
+    host: {
+        class: 'pxb-hero-banner-root', // TODO: Follow ng mat naming conventions
```

Comment:
Can we do the TODO now so we don't have a breaking class change in the future? Is the mat convention to make the host `pxb-hero-banner`?

Reply:
Yes this TODO is a reference to #111 & #112.   

The TODO can actually be removed since the work is already captured in those issues.

Reply:
I'm suggesting we use the desired class name now instead of setting it to `pxb-hero-banner-root` now (breaking change 1) and then changing it to `pxb-hero-banner` later (breaking change 2).

Unless we're planning to use `-root` as our convention for all of the components, in which case it's fine.

Reply:
I guess either way it's a breaking change though...we really just need to do #111 and #112 sooner than later.